Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -237,26 +249,52 @@ To achieve impact at scale, the SCI needs to encourage adoption through ease of
+Because this standard lays out a consequential methodology for calculating the emissions associated with a piece of software, the following must not be included in the calculation:
 
-### Market-based Measures 
-**“Market-based measures"** are achieved through renewables matching (e.g. PPA for solar near a datacenter) that helps lower the grid carbon intensity for everyone in the area. From a carbon accounting perspective PPAs, RECs etc can allow datacenters to get to “100% renewable”, even if the grid is not 100% renewable 24x7 from a grid perspective. Market-based measures includes, but is not limited to the following: 
-- Carbon offsets 
-- Electricity Attribute Certificates (EACs) 
+### Market-based Measures
+**“Market-based measures"** are achieved through renewables matching (e.g. PPA for solar near a datacenter) that helps lower the grid carbon intensity for everyone in the area. From a carbon accounting perspective PPAs, RECs etc can allow datacenters to get to “100% renewable”, even if the grid is not 100% renewable 24x7 from a grid perspective. Market-based measures includes, but is not limited to the following:
+- Carbon offsets
+- Electricity Attribute Certificates (EACs)
 - Power Purchase Agreements (PPAs)
 - Renewable Energy Credits (RECs)
 
-### Infrastructure Measures 
+### Infrastructure Measures
 **“Infrastructure measures”** including any infrastructure that integrate renewables via a "direct wire connection" (e.g. a datacenter with solar panels on the roof and a battery storage located onsite). This is conceptually closer to a Microgrid, where there is a higher % of renewable energy usage than the local grid carbon intensity.
+
+## Reporting Requirements
+
+To comply with this specification and implement carbon transparency, an entity MUST meet these reporting requirements. Reports MUST be made in YAML format, for machine comprehensibility. Additional human readable reports MAY be made and linked to in other formats.
+
+You MAY link to a human readable representation of your report in the further information field, for example, in HTML or PDF format, to provide additional context to your report, or the calculation behind your report.
+
+This human-readable representation SHALL live outside the GSF, and MAY be referenced in the metadata for your report.
+
+### Report structure and required data elements
+The data elements have mandatory key names, which can be found in Appendix A, along with examples of validly formed YAML reporting files .
+#### Mandatory reporting elements
+- Supporting infrastructure and systems contained within the [Software Boundary](#software-boundary).
+- The [Software Carbon Intensity](#reporting-the-sci-value) of the entity itself.
+  - `CI` MUST be reported.
+  - The `R` baseline used MUST be reported.
+  - If the `R` baseline used wa not selected from the [pre-set list](#preset-list-for-baselines) for baselines, then a reason for that choice MUST be provided.
```

Comment:
@jawache this was for the purpose of transparency to the "consumer" of the SCI output. Without this information, it will be difficult to ensure reproducibility and an "honest" assessment of how the final results were arrived at.